.github/workflows/stale.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       pull-requests: write
 
     steps:
-    - uses: actions/stale@v4
```

Comment:
Are there breaking changes between v4 and v8?  Do we need to update our config of the stale bot because of this?

Reply:
No. There are a few "breaking changes", but none of them are significant, given how it's used here:
- [v5](https://github.com/actions/stale/releases/tag/v5.0.0): no changes
- [v6](https://github.com/actions/stale/releases/tag/v6.0.0): when an issue is closed by the bot, it is now closed as "Not planned", not as "Completed"
- [v7](https://github.com/actions/stale/releases/tag/v7.0.0): changes how the bot handles "exempt" issues/PRs, which aren't used here
- [v8](https://github.com/actions/stale/releases/tag/v8.0.0): if there's an error, the Action will fail, not pretend to succeed

Reply:
SGTM; thanks!